changelogs/unreleased/feature-sop-fill-real-values.yml

```diff
@@ -0,0 +1,3 @@
+---
+title : "[SOP] Fill real values"
```

Comment:
Please change to [SOP]: New batch that compute real sales, real manuf and real stock on SOP